src/libs/ga-analytics.js

```diff
@@ -0,0 +1,17 @@
+import ReactGA from 'react-ga4';
+
+const TRACKING_ID = 'G-SDBSSSN65X';
+
+export function init() {
+  // Enable testMode on local or development environment
+  const isDev = !process.env.NODE_ENV || process.env.NODE_ENV === 'development';
```

Comment:
What do you think on including deploy preview (in PRs) as well? So, that the GA events are happening purely from prod users: https://vercel.com/docs/concepts/projects/environment-variables#system-environment-variables.

...unless `NODE_ENV` is really `undefined` in the Vercel build pipeline, but I'm unsure.

```suggestion
  const isDev = !process.env.NODE_ENV || process.env.NODE_ENV === 'development' || process.env.VERCEL_ENV === 'preview';
```

Reply:
I already added two environments on Vercel, `development` for preview and staging, `production` only on prod

Reply:
oh, nice! Sounds good 👍